### PR TITLE
Create animal tables

### PR DESCRIPTION
**Pull request by brahimdidi**
In this project, I used for PostgreSQL to create a database add tables and insert data to it.

## Requirements : 
Create a database named vet_clinic.
Create a table named animals with the following columns
Insert  data into vet_clinics

screenshots 
Screenshot (27)
Screenshot (28)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name like '%mon';
+SELECT name FROM animals WHERE date_of_birth > '2016-01-01' AND date_of_birth < '2019-01-01';
```
Comment (OLIPLICHE) [inline queries.sql]: - [ ] Please you excluded '2016-01-01' and '2019-01-01
- [ ] I suggest this:  `date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-01-01'` (List the name of all animals born between 2016 and 2019)

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+SELECT name FROM animals WHERE neutered = true AND escape_attempts > 3;
```
Comment (OLIPLICHE) [inline queries.sql]: - [ ] Please here is `less than` 3 escape attempts: `escape_attempts < 3` (List the name of all animals that are neutered and have less than 3 escape attempts)

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT name,escape_attempts FROM animals WHERE  weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name != 'Gabumon';
+ SELECT * FROM animals WHERE weight_kg <= 10.4 AND weight_kg <= 17.3;
```
Comment (OLIPLICHE) [inline queries.sql]: - [ ] Please here is `between...`: `weight_kg >= 10.4 AND weight_kg <= 17.3`

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @brahimdidi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights

Commit messages are meaningful 👏 👏
Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please change the title of the Readme to the name of the Project "Vet clinic"
- Please give a better description of the Readme because it is an Introduction to your project


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (brahimdidi) [reply queries.sql]: Ok thanks for the suggestion, I will just use the BETWEEN  clause.

File: queries.sql (hunk 2 above)
Comment (brahimdidi) [reply queries.sql]: That was a mistake, great job noticing that.

File: queries.sql (hunk 3 above)
Comment (brahimdidi) [reply queries.sql]: fixed all that  thanks for the good review.

**Review by keneogu — APPROVED** [review]
Hi @brahimdidi,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**